build.sbt

```diff
@@ -135,10 +136,13 @@ lazy val root: Project = (project in file("."))
 lazy val configTools = (project in file("configTools"))
   .enablePlugins(SbtTwirl)
   .settings(
-    addCompilerPlugin(
-      "org.typelevel" %% "kind-projector" % "0.13.3" cross CrossVersion.full
-    ),
     commonSettings,
+    crossScalaVersions := Seq("2.13.16", scalaVersion.value),
+    // No previous Scala 3 version yet
```

Comment:
It took me a while to figure this out, maybe these comments could be a little more explicit?

Instead of `No previous Scala 3 version yet` perhaps we could say what we're doing here - ensuring binary compatibility during the migration. For the TODO below, in the absence of an existing Scala 3 release to compare against automatically, we're explicitly pointing it at the previous release (`v4.0.0`)?

See what you think though!

Reply:
I agree - it wasn't very clear.  I hope I've improved it in 810cd39edd6fc815f2cbb6d0fbbd9d04b1576a2b